emr-dynamodb-hive/src/main/java/org/apache/hadoop/hive/dynamodb/type/HiveDynamoDBTypeUtil.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+ * except in compliance with the License. A copy of the License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE.TXT" file accompanying this file. This file is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under the License.
+ */
+package org.apache.hadoop.hive.dynamodb.type;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import org.apache.hadoop.hive.serde2.lazy.LazyDouble;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class HiveDynamoDBTypeUtil {
```

Comment:
Could you add some tests for this class?

Reply:
Added tests [here](https://github.com/awslabs/emr-dynamodb-connector/pull/71/files#diff-457b7d995e711ccd87d71de9b6d72d94R17).

Reply:
whoops, sorry, I overlooked those somehow. thanks for pointing them out.